catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/cache/impl/MetacardComparatorTest.java

```diff
@@ -0,0 +1,201 @@
+package ddf.catalog.cache.impl;
+
+import java.net.URI;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.UUID;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.data.impl.MetacardImpl;
+
+public class MetacardComparatorTest {
+
+    MetacardImpl cachedMetacard;
+
+    MetacardImpl latestMetacard;
+
+    Date createdDate;
+
+    Date effectiveDate;
+
+    Date expireDate;
+
+    Date modDate;
+
+    String cachedMetacardId;
```

Comment:
@lessarderic  i need the createdDate, effectiveDate, expireDate, modDate, cachedMetacardId, latestMetacardId as instances because i need to set the attributes on both Metacards to make them the same. Because when I tried just creating a new MetacardImpl, but I noticed that if I used setAttribute, it would change the values on both the cachedMetacard and latestMetacard.